internal/models/test_helpers.go

```
@@ -0,0 +1,95 @@
+func captureOutput(f func()) string {
+	originalStdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	f()
+
+	w.Close()
+	os.Stdout = originalStdout
+
+	var buf bytes.Buffer
+	io.Copy(&buf, r)
```

golangci-lint (1.64.8)

Error return value of `io.Copy` is not checked

(errcheck)